deploy/charts/harvester/values.yaml

```diff
@@ -377,3 +387,8 @@ webhook:
     tag: master-head
 
   httpsPort: 9443
+
+kube-vip:
```

Comment:
will need to add the image of the kube-vip, kube-vip-cloud-provider, and harvester-load-balancer respectively.